src/md4c.c

```diff
@@ -1232,7 +1238,8 @@ md_is_html_any(MD_CTX* ctx, const MD_LINE* lines, int n_lines, OFF beg, OFF max_
  ****************************/
 
 static int
-md_is_hex_entity_contents(MD_CTX* ctx, const CHAR* text, OFF beg, OFF max_end, OFF* p_end)
+md_is_hex_entity_contents(MD_UNUSED MD_CTX* ctx, const CHAR* text,
```

Comment:
the `ctx` argument is unused in several of these internal functions. Rather than removing that argument I thought it would be best to keep it as I'm not sure if there were personal opinions behind uniformity or documentation in this way. Personally I'd remove unused arguments but I leave that up to you @mity